content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -428,7 +422,11 @@ that have a different value and are owned by another manager will result in a
 collaborator's changes. Conflicts can be forced, in which case the value will be
 overridden, and the ownership will be transferred.
 
-It is meant both as a replacement for the original `kubectl apply` and as a
+When fields are removed from a configuration, fields which have no other owners
```

Comment:
Possible reword, something like:
If you remove a field from a configuration, the field manager verifies that this field has no other owners.
If the field is not found in another object, it is deleted from the live object or reset to a default value.

Reply:
Thanks, I went with:

"If you remove a field from a configuration an apply it, server side apply checks
if there are any other field managers that also own the field.  If the field is
not found to be owned by any other field managers, it deleted from the live
object or reset to a default value. The same rule applies to associative list or
map items."